core/server/data/migrations/versions/1.25/1-update-koenig-beta-html.js

```diff
@@ -0,0 +1,59 @@
+const _ = require('lodash');
+const common = require('../../../../lib/common');
+const converters = require('../../../../lib/mobiledoc/converters');
+const models = require('../../../../models');
+
+const mobiledocIsCompatibleWithV1 = function mobiledocIsCompatibleWithV1(doc) {
```

Comment:
Could we make this helper function part of the mobiledoc utility?
The same logic exists in the post model layer.

Reply:
I thought about that but migrations are snapshots in time so I usually inline as much as possible to protect against data changes because I've been burned by before. Helpers change over time to reflect most up-to-date code and data but migrations are designed to be run on old data.